src/serialization/serialization.h

```diff
@@ -213,7 +213,7 @@ inline bool do_serialize(Archive &ar, bool &v)
  * \brief self-explanatory
  */
 #define END_SERIALIZE()				\
-  return ar.stream().good();			\
+  return ar.good();				\
```

Comment:
I personally couldn't care less, but since everybody puts remarks on this, there are trailing spaces at the end of the line here and the other instances of  `if (!r || !ar.good()) return false;`

Reply:
The `\` character is at the end of the line, so this isn't trailing whitespace.

This was formatted by emacs, and the `\` characters lineup in emacs. But not in vim. Running `git blame` + searching for tabs shows that this portion of the file has been formatted like this since 2014. So this maintains the existing style, however frustrating that may be.

Reply:
ok